modui-base.js

```diff
@@ -1,14 +1,18 @@
-var _ = require( 'underscore' );
-var Backbone = require( 'backbone' );
-Backbone.$ = require( 'jquery' );
-var Super = Backbone.View;
-
-var viewOptions = require( 'backbone-view-options' );
-var courier = require( 'backbone-courier' );
-var handle = require( 'backbone-handle' );
-var subviews = require( 'backbone-subviews' );
+( function( root, factory ) {
+	// UMD wrapper
+	if ( typeof define === 'function' && define.amd ) {
+		// AMD
+		define( [ 'underscore', 'backbone', 'jquery', 'backbone.View', 'backbone-view-options', 'backbone-courier', 'backbone-handle', 'backbone-subviews' ], factory );
+	} else if ( typeof exports !== 'undefined' ) {
+		// Node/CommonJS
+		module.exports = factory( require('underscore' ), require( 'backbone' ), require( 'backbone' ).$ , require( 'backbone' ).View, require( 'backbone-view-options' ), require( 'backbone-courier' ), require( 'backbone-handle' ), require( 'backbone-subviews' ) );
```

Comment:
I don't think we need need jquery as a dependency here. we don't use it except for through `this.$el`